### DISCUSSION TITLE
Blank window on `./run ide watch`

### DISCUSSION THREAD

**Discussion by farmaazon** [enso-org/enso#6043] [Help and troubleshooting] [answered]
Despite having d38951d4e986236211dde17ee0e1359c031fc7c6 merged to my branch, I still have an empty electron window. Nothing in web console, the system console shows
```
 INFO ide_ci::program::command: npmℹ️ Resource '/tailwind.css' not found.
 INFO ide_ci::program::command: npmℹ️ Resource '/preload.cjs.map' not found.
 INFO ide_ci::program::command: npmℹ️ Resource '/assets/index.js.map' not found.
```
`./run git-clean` didn't work. The problem is on both `./run ide watch` and `./run ide build`

I guess the fix was tested, so that strange it still reproducible on my side. Any hints?

**Comment by wdanilo** [comment]
@farmaazon I agreed with @PabloBuchu to revert their latests commits ASAP. You'll have that working soon.

**Comment by wdanilo** [comment, marked as answer]
What is the status of this issue @farmaazon and @PabloBuchu ?

**Reply by vitvakatu** [reply]
It was fixed on the develop, @farmaazon confirmed.
